Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3565,3 +3626,321 @@ def remove_tag_from_endpoints_command(client: CoreClient, args: Dict):
     return CommandResults(
         readable_output=f'Successfully removed tag {tag} from endpoint(s) {endpoint_ids}', raw_response=raw_response
     )
+
+
+def parse_list_users(user: dict[str, Any]) -> dict[str, Any]:
```

Comment:
This is only used in `get_list_users_command`, let's use it as an inner method

Reply:
Done